llvm/lib/Target/AMDGPU/DSInstructions.td

```diff
@@ -1268,6 +1279,11 @@ defm DS_PK_ADD_BF16       : DS_Real_gfx12<0x09b>;
 defm DS_PK_ADD_RTN_BF16   : DS_Real_gfx12<0x0ab>;
 defm DS_BPERMUTE_FI_B32   : DS_Real_gfx12<0x0cd>;
 
+defm DS_BVH_STACK_RTN_B32             : DS_Real_gfx12<0x0e0,
+  "ds_bvh_stack_push4_pop1_rtn_b32", true>;
```

Comment:
This adds `ds_bvh_stack_rtn_b32` as an alias for `ds_bvh_stack_push4_pop1_rtn_b32`, right? There should be a test for that if there isn't one already, probably in `gfx12_asm_ds_alias.s`.